README.md

```diff
@@ -86,6 +86,11 @@ If you're using Visual Studio Code for development, it's recommended that you:
 - Install the [Prettier extension](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)
 - [Enable formatting on save](https://scottsauber.com/2017/06/10/prettier-format-on-save-never-worry-about-formatting-javascript-again/). Combined with ESLint and Prettier, this takes care of some code style issues such as formatting, indentation and semicolon consistency for you automatically.
 
+## Git Flow
+*** Make sure that you are familiar with Git Flow before starting! ***
+
+The Team use Git Flow to manage development and release versions, follow the [Git Flow setup steps first](GITFLOW.md).
+
```

Comment:
Suggest it might also be worth adding a step in the 'Setup and first run' section below with the `git flow init` instructions, after the step telling people to clone the repo?  (or at least a link to those instructions?) For new team members it makes sure they don't miss that bit.